### PR TITLE
warning callout component

### PR DESCRIPTION
**Pull request by samMcCarthyBJSS**
Story [AB#10265](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10265)
Task [AB#10769](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10769)

image

### CODE REVIEW

File: app/components/general/components/warning-callout/macro.njk [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{% macro warningCallout(params) %}
+  {%- include './template.njk' -%}
+{% endmacro %}
```
Comment (bjss-chris) [inline app/components/general/components/warning-callout/macro.njk:3]: Just needs a terminating newline.

File: app/components/general/components/warning-callout/settings.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "componentName": "warningCallout",
+    "params": {
+        "heading": "Important",
+        "HTML": "Testing html"
+    }
+}
+
```
Comment (bjss-chris) [inline app/components/general/components/warning-callout/settings.json:8]: Not sure what's going on here but GutHub isn't happy – is there whitespace on the final line?

File: app/components/general/components/warning-callout/template.njk [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  <h3 class="nhsuk-warning-callout__label">
+    <span role="text">
+        <span class="nhsuk-u-visually-hidden">Important: </span>
+        {{ params.heading | safe }}
+    </span>
+  </h3>
+  <div data-test-id="warning-body">
+  {{ params.HTML | safe }}
+  </div>
+</div>
```
Comment (bjss-chris) [inline app/components/general/components/warning-callout/template.njk:11]: Just needs a terminating newline.

**Review by bjss-chris — CHANGES_REQUESTED** [review]
Just a couple end-of-file issues but otherwise looks good.

File: app/components/general/components/warning-callout/settings.json (hunk 2 above)
Comment (samMcCarthyBJSS) [reply app/components/general/components/warning-callout/settings.json:8]: ah there was a space on that one

**Review by bjss-chris — APPROVED** [review]
👍

**Pull request merged**